postgres-types/Cargo.toml

```diff
@@ -48,3 +48,5 @@ uuid-08 = { version = "0.8", package = "uuid", optional = true }
 uuid-1 = { version = "1.0", package = "uuid", optional = true }
 time-02 = { version = "0.2", package = "time", optional = true }
 time-03 = { version = "0.3", package = "time", default-features = false, optional = true }
+
+smol_str = { version = "0.1.23", default-features = false, optional = true }
```

Comment:
This should be `smol_str-01` = { version = "0.1.23", package = "smol_str", default-features = false, optional = true }`.